README.md

```diff
@@ -167,15 +169,18 @@ further evaluation. The allowed fields are described below.
 ### `buildlog_uri` (string) [optional]
 
 URI to the buildlog created during the binary artifact creation.
+The single file at the end of this URI is expected to be GZ-compressed.
```

Comment:
Maybe it could also use the HTTP Transfer-Encoding: gzip header - allows for small transfers with browsers and `curl --compressed`, but can still just view it in the browser, too?

Reply:
Wouldn't that be an implementation detail?

Reply:
Could be a detail that matters. Because if it is foo.txt.gz files, most browsers will not display it, but only offer to download. For machine-processing both are good.

Reply:
I'm in favor of non gzipped files as they are browser viewable.